arxiv_telegram_bot/functions/handlers.py

```diff
@@ -185,7 +185,9 @@ def schedule(update: Update, context: CallbackContext) -> None:
         variable["chat_id"] = chat_id
 
         job_removed = remove_job_if_exists(str(chat_id) + "job", context)
-        # context.job_queue.run_once(updater, int(context.args[0]), context=variable, name=str(chat_id) + 'job')
+        # context.job_queue.run_once(
```

Comment:
Let's remove the comments... Unless we really need them, if we do you can add a `#TODO:`

Reply:
Okay, I'll remove the comments